mlrun/db/httpdb.py

```diff
@@ -3921,16 +3920,20 @@ def patch_model_endpoint(
             attributes_keys=attributes_keys,
             model_endpoint=model_endpoint,
         )
-        response = self.api_call(
+        uid = self.api_call(
             method=mlrun.common.types.HTTPMethod.PATCH,
             path=path,
             params={
                 "attribute-key": attributes_keys,
             },
             body=model_endpoint.json(),
         )
-
-        return mlrun.common.schemas.ModelEndpoint(**response.json())
+        logger.info(
```

Comment:
Do we really need a log of `before` and `after`?

Reply:
what do you think to leave the before or the after log?
I added the after log because now we return None now